repository/boltdb/certificate.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/go-acme/lego/v3/registration"
 )
 
+//Error used internally by Cert(id string). Is not meant to be exposed.
+var certificateNotFound = errors.New("Certificate not found")
```

Comment:
Error variable names should start with "err" or "Err",

Reply:
also, errCertNotFound is verbose enough since we shorten to cert in other names.